app/javascript/scavenger_hunt/index.js

```diff
@@ -1 +1 @@
-export { NavBar } from './components'
+export { NavBar, MicrosoftLoginButton } from './components'
```

Comment:
why export MicrosoftLoginButton if it is only used within NavBar?

Reply:
Oh, you are correct, my bad